starcluster/pysph.py

```diff
@@ -0,0 +1,69 @@
+from starcluster.clustersetup import DefaultClusterSetup
+from starcluster.logger import log
+
+
+class PySPHInstallerBase(DefaultClusterSetup):
+    # PYSPH_PROFILE = '/etc/profile.d/pysph.sh'
+    PYSPH_HOSTS = "/home/pysph/PYSPH_HOSTS"
+
+    def _create_env(self, master):
+        master.ssh.execute(
+            r"""if [ ! -d '/home/pysph/pysph_env' ]; then
+            mkdir '/home/pysph/pysph_env' && virtualenv /home/pysph/pysph_env;
```

Comment:
If you use `virtualenv --system-site-packages` it should let you use the packages installed on the system instead of installing everything via pip.